framework/include/splits/ContactSplit.h

```diff
@@ -31,10 +31,10 @@ class ContactSplit : public Split
  protected:
   std::vector<std::string>              _contact_master;
   std::vector<std::string>              _contact_slave;
-  std::vector<bool>                     _contact_displaced;
+  std::vector<int>                     _contact_displaced;
```

Comment:
Not a big deal, but `unsigned char` or `char` would be a good substitution for Boolean values instead of a 32-bit unsigned int.